exporter/googlecloudexporter/googlecloud.go

```diff
@@ -75,7 +76,7 @@ func generateClientOptions(cfg *Config) ([]option.ClientOption, error) {
 			if cfg.UserAgent != "" {
 				dialOpts = append(dialOpts, grpc.WithUserAgent(cfg.UserAgent))
 			}
-			conn, err := grpc.Dial(cfg.Endpoint, append(dialOpts, grpc.WithTransportCredentials(insecure.NewCredentials()))...)
+			conn, err := grpc.Dial(cfg.Endpoint, append(dialOpts, grpc.WithStatsHandler(&ocgrpc.ClientHandler{}), grpc.WithTransportCredentials(insecure.NewCredentials()))...)
```

Comment:
For my own understanding, can I ask why we haven't used an otel stats handler?

Reply:
This PR is for fixing a missing metrics downstream by our ops-agent: https://github.com/GoogleCloudPlatform/ops-agent/issues/305. This is not a permanent solution, more like a quick walkaround to bring this GA metrics back for our customer to use. 
In the meanwhile, @jsuereth and @punya 's team are working on rewriting this googlecloud exporter. Also side questions for not missing your sentences here: which `otel stats handler` do you mean?

Reply:
I had thought that there was a port from open census into `otel-go`, however, if that is not the case then all is good :) 

I just more wanted to understand if some functionality was missing is all.

Reply:
@sophieyfang is exactly correct, we are working on a reimplementation of this exporter that avoids the OpenCensus dependency. However, that will take longer to complete and test.

Reply:
> @sophieyfang would you mind clarifying in the PR description that you’re adding _back_ metrics that were previously present (and if possible, could you identify the commit or release in which the gRPC metrics went away)?

I have edit the description by linking the PR to ops-agent github issue which contains the details of how it went wrong.